src/modules/75-cf/strings/strings.en.yaml

```
@@ -1600,6 +1602,609 @@ onboarding:
       ```
       cf.close();
       ```
+
+    php: |
+      # Requirements
+
+      Installed <a href="https://www.php.net/">PHP 7.4</a> or a newer version
+      Installed <a href="https://getcomposer.org/">Composer</a>
+
+      # Setup
+      The first step is to install the SDK as a dependency in your application using Composer.
+
+      ```
+      composer require harness/ff-server-sdk
+      ```
+
+      # Code Sample
+      The following is a complete code example that you can use to test the harnessappdemodarkmode Flag you created on the Harness Platform. 
+      When you run the code it will:
+
+      Connect to the FF service.
+      Report the value of the Flag on the webpage. Every time the harnessappdemodarkmode Flag is toggled on or off on the Harness Platform, 
+      the updated value will be updated when the cache refreshes, then refresh the webpage to see the new value.
+      Close the SDK.
+      The example below can also be found in <a href="https://github.com/harness/ff-php-server-sdk/raw/main/online_example.php">online_example.php</a>.
+
+      ```php
+      <?php
+
+      require_once realpath("vendor/autoload.php");
+
+      use Harness\Client;
+      use OpenAPI\Client\Model\Target;
+
+      $SDK_KEY = getenv("SDK_KEY") ?: "";  // you can put your key in env variable or you can provide in the code
```

```suggestion
      $SDK_KEY = getenv({{apiKey}}) ?: "";  // you can put your key in env variable or you can provide in the code
```

If you do this - this will automatically put the generated sdk key into the code snippet to make it easier for user to just copy/paste

---

would be good to add it for every language you've added